src/main/java/seedu/address/ui/cards/PersonCard.java

```diff
@@ -0,0 +1,178 @@
+    @FXML
+    private Label id;
+    @FXML
+    private TextFlow phone;
+    @FXML
+    private TextFlow address;
+    @FXML
+    private TextFlow email;
+    @FXML
+    private FlowPane tags;
+    @FXML
+    private VBox statusBox;
+    @FXML
+    private Label status;
+    @FXML
+    private ImageView calendar;
+    @FXML
+    private Label date;
+
+    /**
+     * Creates a {@code PersonCard} with the given {@code Person} and index to display.
+     */
+    public PersonCard(Person person, int displayedIndex) {
+        super(FXML);
+        this.person = person;
+        initializePersonCard(displayedIndex);
+    }
+
+    /**
+     * Set up the styling of the {@code PersonCard} with {@code displayedIndex}.
+     */
+    private void initializePersonCard(int displayedIndex) {
+        initializeHeader(displayedIndex);
+        initializeBody();
+        initializeTags();
+        initializeStatus();
+        initializeDate();
+    }
+
+    /**
+     * Set up the header of the {@code PersonCard} with {@code displayedIndex}.
+     */
+    private void initializeHeader(int displayedIndex) {
+        id.setText(displayedIndex + ". ");
+        name.setText(person.getName().fullName);
+    }
+
+    /**
+     * Set up the content of the {@code PersonCard}.
+     */
+    private void initializeBody() {
+        setStyling(ATTRIBUTE_PHONE, person.getPhone().value);
+        setStyling(ATTRIBUTE_ADDRESS, person.getAddress().value);
+        setStyling(ATTRIBUTE_EMAIL, person.getEmail().value);
+    }
+
+    /**
+     * Set up the tags of the {@code PersonCard}.
+     */
+    private void initializeTags() {
+        person.getTags().stream()
+            .sorted(Comparator.comparing(tag -> tag.tagName))
+            .forEach(tag -> tags.getChildren().add(new Label(tag.tagName)));
+    }
+
+    private void initializeStatus() {
+        setRandomStatus(); // to be change in when other classes are ready.
+    }
+
+    private void initializeDate() {
+        Image calendarIcon = new Image(this.getClass().getResourceAsStream(CALENDAR_IMAGE_LINK));
+        calendar.setImage(calendarIcon);
+        date.setText("21/07/2020"); // to be changed later!
+    }
+
+    /**
+     * Initialized the GUI of the properties inside {@code PersonCard}.
+     */
+    private void setStyling(String type, String description) {
+        TitleDescription titleDescription = createTitleDescription(type + ": ", description);
+        Text styledTitle = titleDescription.getTitle();
+        Text styledDescription = titleDescription.getDescription();
+        if (type.equals(ATTRIBUTE_PHONE)) {
+            phone.getChildren().addAll(styledTitle, styledDescription);
+        } else if (type.equals(ATTRIBUTE_ADDRESS)) {
+            address.getChildren().addAll(styledTitle, styledDescription);
+        } else if (type.equals(ATTRIBUTE_EMAIL)) {
+            email.getChildren().addAll(styledTitle, styledDescription);
+        } else {
+            assert false;
```

Comment:
Should be an exception instead?

Reply:
hmmm, I am not very sure if it should be an exception instead. I was following the assert from oracle java. 
```
The absence of a default case typically indicates that a programmer believes that one of the cases will always be executed. The assumption that a particular variable will have one of a small number of values is an invariant that should be checked with an assertion.
```

Reply:
I will resolve this conversation since we have concluded it.